examples/peripherals/uart_async_rxtxtasks/main/uart_async_rxtxtasks_main.c

```diff
@@ -0,0 +1,75 @@
+/* UART asynchronous example, that uses separate RX and TX tasks
+
+   This example code is in the Public Domain (or CC0 licensed, at your option.)
+
+   Unless required by applicable law or agreed to in writing, this
+   software is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+   CONDITIONS OF ANY KIND, either express or implied.
+*/
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "esp_system.h"
+#include "esp_log.h"
+#include "driver/uart.h"
+#include "soc/uart_struct.h"
+#include "string.h"
+
+static const int RX_BUF_SIZE = 1024;
+
+#define TXD_PIN (4)
+#define RXD_PIN (5)
+
+void init() {
+    const uart_config_t uart_config = {
+        .baud_rate = 115200,
+        .data_bits = UART_DATA_8_BITS,
+        .parity = UART_PARITY_DISABLE,
+        .stop_bits = UART_STOP_BITS_1,
+        .flow_ctrl = UART_HW_FLOWCTRL_DISABLE,
+        .rx_flow_ctrl_thresh = 122,
```

Comment:
As `.flow_ctrl = UART_HW_FLOWCTRL_DISABLE` and basing on: https://github.com/espressif/esp-idf/blob/de750e99214a51c1bcfdb42b5ada931652a3c531/components/driver/uart.c#L228-L229 I think that the line above may be removed.

Reply:
Thanks, but sorry, maybe I'm missing something here.
Do I understand you correctly, that I could remove line 26: .flow_ctrl = UART_HW_FLOWCTRL_DISABLE,?
If so, and the compiler does not set uninitialized values to 0, and someone changes the uart driver suddenly to do something like (.flow_ctrl == UART_HW_FLOWCTRL_DISABLE), the code might just stop working.

Reply:
Thanks, but sorry, maybe I'm missing something here.
Do I understand you correctly, that I could remove line 26: .flow_ctrl = UART_HW_FLOWCTRL_DISABLE,?
If so, and the compiler does not set uninitialized values to 0, and someone changes the uart driver suddenly to do something like (.flow_ctrl == UART_HW_FLOWCTRL_DISABLE), the code might just stop working.

Reply:
I meant removing line `.rx_flow_ctrl_thresh = 122,`

Reply:
Ah thanks, now it's obvious :)
It's removed in 73bb70d.